checkbox-provider-ce-oem/units/eeprom/test-plan.pxu

```diff
@@ -20,7 +20,8 @@ unit: test plan
 _name: EEPROM auto tests
 _description: Automated EEPROM tests for devices
 include:
-    ce-oem-eeprom/read-write
+nested_part:
+    com.canonical.certification::eeprom-automated
 
```

Comment:
I think we could just remove this test plan and put the `com.canonical.certification::eeprom-automated` test plan into ce-oem-automated directly